src/app/Components/Home/index.snap.test.js

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { BrowserRouter as Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
-import { createStore, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
-import logger from 'redux-logger';
+import configureStore from 'redux-mock-store';
```

Comment:
I think you missed committing the change where you add this dependency to the `package.json`